### PR TITLE
iOS start from library

### PR DESCRIPTION
**Pull request by guperrot**

### CODE REVIEW

File: AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m [hunk 1]
```diff
@@ -18,16 +19,24 @@ + (NSString *) getAppSecret
     NSString * plistPath = [[NSBundle mainBundle] pathForResource:@"AppCenter-Config" ofType:@"plist"];
     NSDictionary * config = [NSDictionary dictionaryWithContentsOfFile:plistPath];
 
-    appSecret = [config objectForKey:@"AppSecret"];
     // If the AppSecret is not set, we will pass nil to MSAppCenter which will error out, as expected
-  }
+    appSecret = [config objectForKey:@"AppSecret"];
 
+    // Read start automatically flag, by default it's true if not set.
```
[the remark this answers is not in the document]
Comment (jwallra) [reply AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m]: Nevermind, midread the sentence.  Looks fine

File: AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m [hunk 2]
```diff
@@ -5,6 +5,7 @@
 @implementation AppCenterReactNativeShared
 
 static NSString *appSecret;
+static BOOL startAutomatically;
 static MSWrapperSdk * wrapperSdk;
```
Comment (dhei) [inline AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m]: nit - `MSWrapperSdk *wrapperSdk`

Comment (guperrot) [reply AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m]: Will include it in my next PR that I am working on as this is existing code ;)

File: AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m [hunk 3]
```diff
@@ -18,16 +19,24 @@ + (NSString *) getAppSecret
     NSString * plistPath = [[NSBundle mainBundle] pathForResource:@"AppCenter-Config" ofType:@"plist"];
     NSDictionary * config = [NSDictionary dictionaryWithContentsOfFile:plistPath];
 
-    appSecret = [config objectForKey:@"AppSecret"];
     // If the AppSecret is not set, we will pass nil to MSAppCenter which will error out, as expected
```
Comment (dhei) [inline AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m]: We should update this comment. Nil app secret is a valid scenario and MSAppCenter won't error out.

Comment (guperrot) [reply AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m]: We don't need any comment actually now, removed on another branch.

**Pull request merged**